### PR TITLE
Add map of Thailand

### PR DESCRIPTION
**Pull request by tae8838**

### CODE REVIEW

File: packages/thailand/thailand.svg [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+<!-- Created for MapSVG plugin: http://mapsvg.com -->
+<svg
+   xmlns:mapsvg="http://mapsvg.com"
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   mapsvg:geoViewBox="97.344728 20.463430 105.640023 5.614417"
+   width="559.57092"
+   height="1024.7631">
```
Comment (VictorCazanave) [inline packages/thailand/thailand.svg]: Could you remove the unnecessary attributes and set the `viewBox` please? You can look at other SVG maps

File: packages/thailand/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+### yarn
+
+`yarn add @svg-maps/thailand`
+
+## Credits
+
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/thailand) and free to be used in any custom project.
```
Comment (VictorCazanave) [inline packages/thailand/README.md]: Could you add the changes made on the original map please?

**Comment by tae8838** [comment]
@VictorCazanave Thanks for the review! Updated. Hope its all good now!

**Comment by VictorCazanave** [comment]
@tae8838 That's perfect! Thanks again 😃

**Comment by VictorCazanave** [comment]
@all-contributors please add @tae8838 for code

**Pull request merged**